### DISCUSSION TITLE
What is your work flow when you want to resume from certain point from the already writen script

### DISCUSSION THREAD

**Discussion by LintaoAmons** [getgauge/taiko#2680] [General]
- So, I have wrote down a script and I want to stop(breakpoint) at somewhere and continue REPL mode
- I'm already using REPL mode to open a browser and explore it manually, at some point, I want to copy some step I have already written in the past, and I want to copy those steps into REPL to execute.

How you deal  with those scenarios?

**Comment by zabil** [comment]
Have you tried the [`repl`](https://docs.taiko.dev/api/repl/) command?

**Reply by LintaoAmons** [reply]
What do you mean by `repl` command?  I didn't see there's option, only `-o`, `-l`, etc.

**Reply by zabil** [reply]
https://docs.taiko.dev/api/repl/

**Reply by LintaoAmons** [reply]
Thx!~
